server/mix.exs

```diff
@@ -1,3 +1,4 @@
+:public_key.cacerts_load("/Users/Shared/CAINLROOT_B64.crt")
```

Comment:
need to comment this out be default

Reply:
pushed updates to this branch from my fork, should be fixed!